themes/digital.gov/layouts/shortcodes/job-board.html

```diff
@@ -0,0 +1,72 @@
+{{/* <!-- prettier-ignore -->
+  Communities of Practice Job Board Shortcode
+  Formats job posting information from "job_board.yml" into a table, as seen on Communities of Practice Job Board "job-board.md"
+  Add new job posting on "job_board.yml" to display in job posting table.
+  No params needed
+*/}}
+
+{{/* Dictionary used to convert CoP names to its acronym for style formatting */}}
+{{ $acronymToCop := dict
+  "Communicators" "FCN"
+  "Multilingual" "ML"
+  "Plain Language" "PL"
+  "Social Media" "SM"
+  "User Experience" "UX"
+  "Web Analytics" "WAO"
+  "Web Managers" "WCM"
+  "Contact Center" "CC"
+}}
+
+{{ $now := now.Format "2006-01-02" }}
+{{ $anyActivePost := false }}
+
+
+<!-- This formats job postings into the DG blog style -->
+{{ range .Site.Data.job_board }}
+  {{ if ge .close_date $now }}
+    {{ $anyActivePost = true }}
+    <article class="card card-job" aria-labelledby="{{ .title }}">
+      <div class="card-job__content">
+        <h3 id="{{ .title }}">{{- .title | markdownify -}}</h3>
+        <div class="card-job__summary">
+          <div class="card-job__department">
+            {{ .department | markdownify }}
+          </div>
+          <div class="card-job__location">{{ .location | markdownify }}</div>
+          <p>{{ .description | markdownify }}</p>
+        </div>
+      </div>
+      <div class="card-job__apply">
+        <a
```

Comment:
We should set the `aria-label` so screen readers will read `Apply for job title goes here`.

```html
aria-label="Apply for  {{ .title }}"
```